packages/contractkit/src/base.ts

```diff
@@ -32,6 +32,7 @@ export const ProxyContracts = [
   'BlockchainParametersProxy',
   'DoubleSigningSlasherProxy',
   'DowntimeSlasherProxy',
+  'DowntimeSlasherIntervalsProxy',
```

Comment:
No, basically the text to search it will be downtimeSlasher
Same with the registry. I think that it's better to have the same generic name for that kind of contract, without thinking in the version or implementation

One thing that I don't know if we should do, is to remove the old DowntimeSlasher implementation.

Because, if that's the case, we could leave the same names here, but only generate the new `.sol` into the `DowntimeSlasher` contract.
This way, only the generator knows which version (or which implementation) will be that contract implementation for the kit (don't know if it makes sense)

Reply:
Removed with refactor